db/migrate/20160324225152_rename_table_orders_to_reservations.rb

```diff
@@ -0,0 +1,5 @@
+class RenameTableOrdersToReservations < ActiveRecord::Migration
+  def change
+    rename_table :orders, :reservations
```

Comment:
nice! I see that we are really making us of "pivoting" old code into what will now help us for our current project!